features/input-color.yml.dist

```diff
@@ -0,0 +1,14 @@
+# Generated from: input-color.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
```

Comment:
BCD says Firefox Android is only partial b/c only allows preset colors not custom ones. Weird, but ok.